app/models/fact_names/foreman_ansible/fact_name.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module ForemanAnsible
+  # Define the class that fact names that come from Ansible should have
+  # It allows us to filter facts by origin, and also to display the origin
+  # in the fact values table (/fact_values)
+  class FactName < ::FactName
+    def origin
+      'Ansible'
+    end
+
+    def icon_path
+      'Ansible.png'
+    end
```

Comment:
Can you create a TODO item (Redmine perhaps): Move images from app/assets/images/xyz.png into icons16x16 and icons64x64 subdirectories? We now have two sizes of images which allows us to show big images. We do not do this yet, but we can in the future.